benchmarks/user/benchmark.py

```diff
@@ -47,6 +47,9 @@ def default_list(ctx, param, value):
                      callback=default_list, help='Time order of the simulation'),
         click.option('-t', '--tn', default=250,
                      help='End time of the simulation in ms'),
+        click.option('-a', '--autotune', default='aggressive',
```

Comment:
now that I think about it, having moved `-a` from `option_performance` to `option_simulation` doesn't make any sense